open-feature/openfeature.dev
# feat: Add Flipt Web Provider to ecosystem

## This PR


This commit adds the official Flipt OpenFeature Provider that uses the OpenFeature Web SDK.

Official docs with reference could be found at:
- https://docs.flipt.io/integration/openfeature

### Related Issues


No related issue, prepared PR right away.

### Notes


Originally asked in CNCF Slack's channel `#openfeature` if this PR can be prepared:
https://cloud-native.slack.com/archives/C0344AANLA1/p1722603297597369?thread_ts=1722590114.396409&cid=C0344AANLA1

### Follow-up Tasks




### How to test


1. `make start` or `yarn start`
2. Open `http://localhost:3000/ecosystem?instant_search%5BrefinementList%5D%5Bvendor%5D%5B0%5D=Flipt`
3. See if there is `Flipt JavaScript Web Provider` and if it redirects to correct web page.

<details>
<summary>Screenshot of the page from the step 2.</summary>

> ![Screenshot 2024-08-06 at 21 05 52](https://github.com/user-attachments/assets/6221caf9-97d6-4107-9ba1-547e9e90c19d)

</details>
